package com.sky.task;

import lombok.extern.slf4j.Slf4j;
import org.springframework.scheduling.annotation.Scheduled;
import org.springframework.stereotype.Component;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.util.Date;

@Component
@Slf4j
public class MyTask {


    //@Scheduled(cron = "0/5 * * * * ? ")
    public void print(){
        log.info("定时任务启动"+new Date());
        log.info("定时任务结束"+ LocalDateTime.now());
    }
}
